libs/wingc/src/jsify.rs

```diff
@@ -395,7 +404,7 @@ impl<'a> JSifier<'a> {
 					.collect::<Vec<String>>()
 					.join(", ");
 
-				if is_mutable_collection(expression) {
+				if is_mutable_collection(expression) || *self.is_json.borrow() {
```

Comment:
Why special treatment for json here? Shouldn't this be related to whether this is a `MutJson` or not and not generally a json?

Reply:
So here was the thinking..  the JsonLiteral will take care of adding the `Object.freeze()` or not at the root level. This way json elements will inherit their mutability from the JsonLiteral.

so that
```js
let x = MutJson {
 	y: {
		z: [1, 2, 3]
	}
}

let a = MutJson {
 	b: {
		c: [1, 2, 3]
	}
}

// Should jsify to
const x = {"y":{"z":[1, 2, 3]}};
const a = Object.freeze({"b":{"c":[1, 2, 3]}});
```